docs/cloud-native-security/session-view.asciidoc

```diff
@@ -92,6 +91,11 @@ Session View includes additional badges not pictured above:
 //[role="screenshot"]
 //image::images/session-view-script-button.png[The Script button]
 
+* The alert badge for multiple alerts. If a single process event or its children cause alerts of multiple types, multiple icons appear on the alert badge (a gear, a page, and a globe for `process`, `file`, and `network` alerts, respectively):
++
+[role="screenshot"]
+image::images/session-view-alert-types-badge.png[The alert badge for a command with all three alert types,width=55%,height=55%]
+
```

Comment:
I'm not sure if we should mention the "or its children" part. I believe they are only for the respective process  (pinging @Omolola-Akinleye to confirm that)

What about adding a reference that each Icon relates to the respective [Event Collection](https://www.elastic.co/guide/en/security/master/configure-endpoint-integration-policy.html#event-collection), and while the gear will always be present (since session data requires process event collection), file and network event collection need to be enabled to have multiple alert types.